src/api/history.c

```diff
@@ -18,8 +18,6 @@
 #include "overTime.h"
 // config struct
 #include "config/config.h"
-// read_setupVarsconf()
-#include "config/setupVars.h"
```

Comment:
Unnecessary here, found when `grep`ing for `read_setupVarsconf()` to see where it is used